### ISSUE TITLE
path resolver is broken on nesting rust project

### ISSUE THREAD

**Issue by oligamiq** [burtonageo/cargo-bundle#170]
```bash
$ cargo bundle --target x86_64-apple-ios --format ios
    Finished dev [unoptimized + debuginfo] target(s) in 0.08s
warning: iOS bundle support is still experimental.
    Bundling ios-beta.app
error: Failed to copy binary from "/Users/runner/work/nickname/nickname/nickname/target/x86_64-apple-ios/debug/cacao"
  Caused by: "/Users/runner/work/nickname/nickname/nickname/target/x86_64-apple-ios/debug/cacao" does not exist
$ echo $(pwd)
/Users/runner/work/nickname/nickname/nickname/examples/iphone
```
![image](https://github.com/burtonageo/cargo-bundle/assets/69239094/1b505410-3a63-4618-bd22-3430cbc26f0e)

I changed `examples` -> `example`  -> `test`, but all folder is failed.
I cp `../` but failed.
I cp without `nickname` workspace and succussed

I think path resolver is broken on nesting rust project.

**Comment by radiohertz** [comment]
Is this fixed? `v0.6.1` fails with "Failed to copy binary from " but bundling with `v0.6.0` works fine. 

cc @ActuallyHappening @mdsteele

**Comment by mdsteele** [comment]
If you can diagnose this and send a PR to fix, I can put out a `v0.6.2` to include it.
